pages/common/ssh-keygen.md

```diff
@@ -33,3 +29,7 @@
 - Change the password of a key:
 
 `ssh-keygen -p -f ~/.ssh/{{filename}}`
+
+- Change the type of the key format (for example from OPENSSH format to PEM), the file will be rewritten in-place:
```

Comment:
I'm not sure the `the file will be rewritten in-place` is neccesary, what do you think?

Reply:
could be removed , just added it as a _heads up_